src/nspawn/nspawn.c

```diff
@@ -187,6 +187,7 @@ static uid_t arg_uid_shift = UID_INVALID, arg_uid_range = 0x10000U;
 static bool arg_userns_chown = false;
 static int arg_kill_signal = 0;
 static bool arg_unified_cgroup_hierarchy = false;
+static bool arg_cgroup_namespace = false;
```

Comment:
I think this should be unconditionally enabled if available. (or maybe only configurable via some env var or so, for debugging purposes only.) This appears something we would always want if it works.